heroku_connect/db/models/base.py

```diff
@@ -34,6 +48,7 @@ def __new__(mcs, name, bases, attrs):
             new_class._meta.local_fields.remove(is_deleted)
 
         new_class._meta.managed = False
+        new_class.add_to_class('objects', HerokuConnectModelManager())
```

Comment:
I think you want this to be `_default_manager`. I am not sure tho, you might want to investigate this further.